lib/dynamoid/associations/belongs_to.rb

```diff
@@ -39,8 +39,14 @@ def associate(hash_key)
       #
       # @since 0.2.0
       def target_association
-        has_many_key_name = options[:inverse_of] || source.class.to_s.underscore.pluralize.to_sym
```

Comment:
In every other file the `pluralize` comes before the `underscore`, so I changed this to be consistent